### PR TITLE
Edited the Product Planning according to TA Feedback fixes #45,fixes #36 and fixes #58.

### LINKED ISSUE #36: Glossary for Product Planning

### LINKED ISSUE #45: Process feedback Product Planning

### LINKED ISSUE #58: Adjust roadmap according to monday meeting
Adjust roadmap according to monday meeting with small tweaks to upcoming weeks.

### PR DESCRIPTION
**Pull request by eanker**
We might find a better front-cover image, but that we shall decide together.

### CODE REVIEW

File: Documentation/General/productPlanning/requirements.tex [hunk 1]
```diff
@@ -118,17 +107,17 @@ \subsection{Non-functional requirements}
 
 \item Git shall be used with the PR methodology: Every new feature will be developed in a branch, once completed it will be turned into a pull request. This will be reviewed and improved, and if it is accepted the branch gets merged into master.
 
-\item Frontend and Backend shall communicate in a RESTful way, using an API.
+\item Front-end and Back-end shall communicate in a RESTful way, using an API.
```
Comment (jwgmeligmeyling) [inline Documentation/General/productPlanning/requirements.tex]: It should be `Frontend` and `backend` without `-`

File: Documentation/General/productPlanning/requirements.tex [hunk 2]
```diff
@@ -118,17 +107,17 @@ \subsection{Non-functional requirements}
...
 
-\item The product shall consist of a seperate frontend and backend repo, ensuring proper RESTful implementation is used.
+\item The product shall consist of a separate front end and backend repo, ensuring proper RESTful implementation is used.
```
Comment (jwgmeligmeyling) [inline Documentation/General/productPlanning/requirements.tex]: and here `frontend` as well ;)

Comment (eanker) [inline Documentation/General/productPlanning/requirements.tex]: I will edit all the frontends and backends.

File: Documentation/General/productPlanning/planning.tex [hunk 3]
```diff
@@ -34,32 +34,34 @@ \section{Roadmap}
...
 \item Choose songs to use as test data.
 \item Create testcases to measure the performance of our algorithm.
-\item Create initial version of algorithm.
+\item Design the neural network. 
+\item You can vote how good or bad a song is.
 \end{itemize}
 
 \item Week 5: Sprint 4\\
-The music-matching algorithm will be extended.
+The connection between the front and back-end will be adjusted. We will get an algorithm that does the mood detection for us.
```
Comment (jwgmeligmeyling) [inline Documentation/General/productPlanning/planning.tex]: > We will get a basic overview of to to create an algorithm that will provide song suggestions for the room.

File: Documentation/General/productPlanning/planning.tex [hunk 4]
```diff
@@ -34,32 +34,34 @@ \section{Roadmap}
...
-\item Assess the quality of the initial algorithm.
-\item Incorporate user feedback in the algorithm.
+\item Make API's for the front-end (chat and general room).
+\item Connection between API's and back-end.
+\item Connect an external libary or API to have an mood-detection algorithm.
 \end{itemize}
 
 \item Week 6: Sprint 5\\
-The music-matching algorithm will be tweaked and extended. We also review our own code and test quality, since we're half way.
+The music-matching algorithm will be made. We also review our own code and test quality, since we're half way.
```
Comment (jwgmeligmeyling) [inline Documentation/General/productPlanning/planning.tex:61]: > - User votes will improve the suggestions over time.

**Comment by TimvdLippe** [comment]
Reminder: this should be delivered on BlackBoard at Upload.

File: Documentation/General/productPlanning/planning.tex [hunk 5]
```diff
@@ -126,36 +125,29 @@ \section{User stories of features}
...
-The system also has to be manually tested for the main interaction with the system. Lastly, integration tests are created and ran succesfully.\\
+The system also has to be manually tested for the main interaction with the system. Lastly, integration tests are created and ran successfully.\\
 
 The final product is done when all requirements are fulfilled and the implementation matches the specification. The implementation should be well tested using unit tests and preferably integration / end-to-end tests as well.
 If a should-have requirement is not implemented, this has to be agreed upon by all team members with a valid reason, and the product should still be usable without the feature.
-Code is consistent, validated by the SIG test and documented.
-
-\chapter{Glossary}
\ No newline at end of file
+Code is consistent, validated by the \gls{SIG} test and documented.
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/planning.tex]: 'The code base is formatted and documentated in a consistent manner and validated by the SIG test.'

File: Documentation/General/productPlanning/planning.tex [hunk 6]
```diff
@@ -126,36 +125,29 @@ \section{User stories of features}
...
 \section{User stories of know-how acquisition}
+As a developer, \\
+When I do not know something, \\
+Then I will search on the internet for relevant information.
 
 \section{Initial release plan}
-
+%3.5. Initial release plan (milestones, MRFs per release)
+Our milestones consists only of the sprint number .
+For example, if a feature shall be made in Sprint 3 it has milestone Sprint 3.
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/planning.tex]: 'Each sprint has a corresponding milestone. For example, if a feature will be developed during sprint 3, the corresponding GitHub issue will have milestone 3. This makes sure that the roadmap outlines are closely followed and progress can be easily determined by using the GitHub tools.'

**Comment by TimvdLippe** [comment]
Ready to rumble. You will upload it @Azkle ?

**Comment by jwgmeligmeyling** [comment]
Can be merged! :+1:

**Pull request merged**